packages/types/src/index.ts

```diff
@@ -1 +1,5 @@
+export * from "./event-bus"
```

Comment:
suggestion: Maybe to avoid any collision if multiple domain exports the same named type, we should export such as `export * as EventBus from "./event-bus"` ? When you import, you would import `EventBus` and then use for example `EventBus.Subscriber`. wdyt?